tracer/src/Datadog.Trace/Activity/OtlpHelpers.cs

```diff
@@ -247,7 +247,16 @@ internal static void SetTagObject(Span span, string key, object? value)
                     span.SetMetric(key, us);
                     break;
                 case int i: // TODO: Can't get here from OTEL API, test with Activity API
-                    span.SetMetric(key, i);
+                    // special case where we need to remap "http.response.status_code"
+                    if (key == "http.response.status_code")
```

Comment:
do we need to worry about casing here?

Reply:
I am unsure - the spec doesn't seem to be entirely clear on casing - but I think it is supposed to be lower cased

@zarirhamza Any ideas whether other tracers are handling casing?

Reply:
From @zarirhamza yesterday - we follow the OpenTelemetry spec which specifies the key to be lowercase so that is all we check for and expect, so no need to worry about casing here.

This does mean that if (for some reason) the casing was incorrect it wouldn't get mapped correctly.